app/static/js/createEvents.js

```diff
@@ -17,11 +16,20 @@ function updateDate(obj) {
   }
 }
 
+function updateTime(obj) {
+  if (obj.id == "startTime") {
+    $("#startTime").timepicker({timeFormat: 'h:mm p',});
+  }
+
+  if (obj.id == "startTime") {
+    $("#startTime").timepicker({timeFormat: 'h:mm p',});
+  }
```

Comment:
These are the exact same conditionals.

Reply:
Aren't you already initializing these when the document loads?

Reply:
removed this function since we already initialized when document laods.